### PR TITLE
Fix ring job

### PR DESCRIPTION
**Pull request by cschwede**
Fixes a possible race condition and two minor issues.

### CODE REVIEW

File: pkg/swiftring/funcs.go [hunk 1]
```diff
@@ -60,6 +60,10 @@ func DeviceList(ctx context.Context, h *helper.Helper, instance *swiftv1beta1.Sw
 			capacity := resource.MustParse(storageInstance.Spec.StorageRequest)
 			weight, _ := capacity.AsInt64()
 			if err == nil {
+				if foundClaim.Status.Phase != "Bound" {
```
Comment (stuggi) [inline pkg/swiftring/funcs.go:63]: could use available const https://pkg.go.dev/k8s.io/api@v0.28.7/core/v1#PersistentVolumeClaimPhase
`"Bound"` -> `corev1.ClaimBound`

File: pkg/swiftring/funcs.go [hunk 2]
```diff
@@ -61,6 +60,10 @@ func DeviceList(ctx context.Context, h *helper.Helper, instance *swiftv1beta1.Sw
...
+					err = fmt.Errorf("PVC %s found, but not bound yet (%s). Requeueing", cn, foundClaim.Status.Phase)
+					return "", "", err // requeueing
+				}
 				capacity := foundClaim.Status.Capacity["storage"]
```
Comment (stuggi) [inline pkg/swiftring/funcs.go:67]: `"storage"` => `corev1.ResourceStorage`

File: pkg/swiftring/funcs.go (hunk 1 above)
Comment (cschwede) [reply pkg/swiftring/funcs.go:63]: Done

File: pkg/swiftring/funcs.go (hunk 2 above)
Comment (cschwede) [reply pkg/swiftring/funcs.go]: Done

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**